### PR TITLE
Added spanish translation

### PR DESCRIPTION
**Pull request by darltrash**
*Thank you for contributing to ProtonPlus! So that your Pull Request can be handled effectively, please populate the following fields (delete sections that are not applicable)*

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (Newbytee) [inline .vscode/settings.json:1]: Are you sure about adding your `.vscode` folder as part of this?

Comment (Vysp3r) [reply .vscode/settings.json:1]: Yeah, it's exactly what I was gonna say.

Comment (darltrash) [reply .vscode/settings.json:1]: > Are you sure about adding your `.vscode` folder as part of this?

HAHAAHHA DAMN FORGOT :laughing:

Comment (darltrash) [reply .vscode/settings.json]: whoops, fixed now :)

**Review by Vysp3r — APPROVED** [review]

**Comment by Vysp3r** [comment]
Was the translation working on your computer?

**Comment by darltrash** [comment]
> Was the translation working on your computer?

Yeah pretty much

**Comment by Vysp3r** [comment]
Okay because I wasn't able to make it work using 'export LANG=es_ES && run com.vysp3r.ProtonPlus' it does not seem to work.

**Pull request merged**